Upstream-Status: Accepted [Expected to be included in parted versions > 3.1]
Upstream-URL: http://git.savannah.gnu.org/cgit/gnulib.git/patch/?id=cbc11ff0020eb9c04caea6b3e7dc4e4281dff1f9

From cbc11ff0020eb9c04caea6b3e7dc4e4281dff1f9 Mon Sep 17 00:00:00 2001
From: Andreas Oberritter <obi@opendreambox.org>
Date: Sun, 18 Mar 2012 17:39:14 +0000
Subject: git-version-gen: don't let "prefix" envvar cause trouble

* build-aux/git-version-gen (prefix): Initialize properly,
so as not to use a value specified via the environment.
Details here: http://thread.gmane.org/gmane.comp.gnu.parted.bugs/10810
---
diff --git a/build-aux/git-version-gen b/build-aux/git-version-gen
index d5542a2..0fa9063 100755
--- a/build-aux/git-version-gen
+++ b/build-aux/git-version-gen
@@ -1,6 +1,6 @@
 #!/bin/sh
 # Print a version string.
-scriptversion=2012-01-06.07; # UTC
+scriptversion=2012-03-18.17; # UTC
 
 # Copyright (C) 2007-2012 Free Software Foundation, Inc.
 #
@@ -92,6 +92,8 @@ Options:
 
 Running without arguments will suffice in most cases."
 
+prefix=v
+
 while test $# -gt 0; do
   case $1 in
     --help) echo "$usage"; exit 0;;
@@ -120,7 +122,6 @@ if test -z "$tarball_version_file"; then
 fi
 
 tag_sed_script="${tag_sed_script:-s/x/x/}"
-prefix="${prefix:-v}"
 
 nl='
 '
--
cgit v0.9.0.2
